FirebaseRemoteConfigSwift/Tests/SwiftAPI/APITestBase.swift

```diff
@@ -22,30 +27,54 @@ class APITestBase: XCTestCase {
   static var mockedFetch: Bool!
   var app: FirebaseApp!
   var config: RemoteConfig!
+  var console: RemoteConfigConsole!
   var fakeConsole: FakeConsole!
 
   override class func setUp() {
     if FirebaseApp.app() == nil {
+      FIROptionsMock.mockFIROptions()
       FirebaseApp.configure()
```

Comment:
Hmm do you know why we aren't just using `configure(options:)` and passing that in instead of mocking things?